docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,374 @@
+The list will evolve depending adoption.
+
+Example of well-known annotations, adding cosign signatures to an image:
+```json
+{
+   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+   "digest": "sha256:c3c58223e2af75154c4a7852d6924b4cc51a00c821553bbd9b3319481131b2e0",
+   "size": 528,
+   "platform": {
+      "architecture": "arm64",
+      "os": "linux"
+   },
+   "annotations": {
+      "dev.cosignproject.cosign/signature.e59879c": "...",
+      "dev.sigstore.cosign/bundle.6436749": "...",
+      "dev.sigstore.cosign/certificate.776a3f1": "...",
+      "dev.sigstore.cosign/chain.8adf60e": "...",
+      "dev.cosignproject.cosign/signature.de25bf6": "...",
+      "dev.sigstore.cosign/bundle.37d34ff": "...",
+      "dev.sigstore.cosign/certificate.e8adf60": "...",
+      "dev.sigstore.cosign/chain.b1c64a3": "...",
+      "org.opencontainers.signatures.index":"dev.cosignproject.cosign/signature.e59879c,dev.cosignproject.cosign/signature.de25bf6,dev.sigstore.cosign/bundle.37d34ff,dev.sigstore.cosign/certificate.e8adf60,dev.sigstore.cosign/chain.b1c64a3,dev.sigstore.cosign/bundle.6436749,dev.sigstore.cosign/certificate.776a3f1,dev.sigstore.cosign/chain.8adf60e",
+   }
+}
+```
+
+### Scenarii
+
+The current proposal doesn't change the workflow for [the Refrigerated Truck Driver 🚚](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md#3-the-refrigerated-truck-driver-), [the Bodega Owner 🏪](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md#4-the-bodega-owner-) and the [The Ice Cream Lover 😍](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md#5-the-ice-cream-lover-) personas.
+Two scenarii are identified for the [Ice Cream Factory Worker 🏭](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md#2-the-ice-cream-factory-worker-) and [Health Inspector 🕵️‍♀️](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md#2-the-ice-cream-factory-worker-) personas.
+
+1/ The 🏭 wants to enhance an existing image without changing its digest, already consumed by 😍, so 🕵️ can validate it.
+2/ The 🏭 pushes an image with all the pieces needed by 🕵 without changing 😍 workflow.
+
+#### Enhance existing image
+
+An image is already pushed and used. To add artifacts and annotations to it without re-pushin and changing the known digest, a new OCI index is pushed on a new tag, referencing the original image.
+The tag **SHOULD** have the following format:
+`<repo>:<alg>-<ref>.<hash>`:
+- E.g. `registry.example.org/project-d:sha256-0000000000000000000000000000000000000000000000000000000000000000.0404040404040404`
+- `<alg>`: the digest algorithm
+- `<ref>`: the referenced digest (limit of 64 characters)
+- `<hash>`: hash of this artifact (limit of 16 characters)
+
+If the referenced image is an OCI Index, the manifests **SHOULD** be inserted first, for backward compatibility with runtimes.
```

Comment:
```suggestion
If the referenced image is an OCI Index, the manifests **MAY** be inserted first, for backward compatibility with runtimes.
```

I would change this to MAY as it's up to the people creating the artifacts if they want to provide backward compatibility or not.